stage1/init/init.go

```diff
@@ -171,6 +172,50 @@ func machinedRegister() bool {
 	return found == 2
 }
 
+func lookupPath(bin string, paths string) (string, error) {
+	pathsArr := strings.Split(paths, ":")
```

Comment:
*Gasp!* You are hardcoding colon! What about Windows? They use semicolons for separating paths, no?

Just joking, we probably don't care about Windows. Do we?

But if you care, then I would recommend you looking at path/filepath's SplitList.